src/pages/Swap/AdvancedTradingViewBox/AdvancedTradingView.tsx

```diff
@@ -16,10 +16,8 @@ export function AdvancedTradingViewBox({ children }: { children: React.ReactNode
   const isInProMode = pathname.includes('/pro')
```

Comment:
also this isInPro mode should check in local storage not in the path since its linked to localStorage state

Reply:
Will check if we can remove this. Because there is a view state and a user settings state. For example, this edge case should be possible: go to swapr.eth.limo/#/swap, view swapbox in normal mode but still have the pro mode preference. So there is a difference between user preference and chosen route. Maybe we can derive the view state from the route  and be cleaner :)

Reply:
I took a look into this and I think we need to have both ways. One for visual state and other for user settings state, which controls the application flow.